### DISCUSSION TITLE
ISSUE

### DISCUSSION THREAD

**Discussion by CinekPoland** [OpenRadioss/OpenRadioss#913] [Q&A] [answered]
Hi.  I installed openradioss on Windows 10 for workstastion version, also I installed  visual studio 2022, IntelONEAPI basee toolkit, intelONEAPI HPC toolkit , and microsoft MPI, I tried run cell phone drop test and also bumper simulation, and I got this error. Doeas anyone knows how to fix it ?
![207988813-9d71731e-2c45-4313-90b7-cb79b142afba](https://user-images.githubusercontent.com/119892611/208170611-1a6594b2-14f2-4a76-9e79-f351e07e2bb4.jpg)

**Comment by MarianBulla** [comment, marked as answer]
Hello,
please review "How to run OpenRadioss":
![image](https://user-images.githubusercontent.com/105412502/208184867-54a0ab29-4443-4c6a-8839-00c9dd1fdfa2.png)

You need to set the environmental path variables in the command window, pointing to the neccessary libraries.
Hope it helps you ?
Thanks and best regards,
Marian

**Comment by CinekPoland** [comment]
Thank you, I corrected environment variables settings and open radios works very well.
